src/presentation/pages/Join.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <h1>Team</h1>
+  <Section :title="t(message)" />
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue';
+import { Section } from 'codex-ui/vue';
+import useNoteSettings from '@/application/services/useNoteSettings.ts';
+import { useHead } from 'unhead';
+import { useI18n } from 'vue-i18n';
+import { useRoute } from 'vue-router';
+
+const route = useRoute();
+const { t } = useI18n();
+
+const message = ref<string>('');
+
+async function checkJoinEligibility() {
+  useNoteSettings().joinNote(route.params.id as string)
+    .then(() => {})
+    .catch(err => message.value = err.message);
+}
+
+checkJoinEligibility();
+
+useHead({
+  title: t('join.title'),
+});
```

Comment:
it is a technical route, ideally user should now see it, so the title is redundant